Gaijinizer patch/Map120.txt

```diff
@@ -192,9 +195,10 @@ somewhere ...#MTLed
 だんだん武器を持った怖そうな人間が集まってきて
 出るに出られなくなってきたんだよね……
 > CONTEXT: Map120/events/5/pages/1/44/Dialogue/enemy11
-Well, I was waiting to come back soon
-A scary human with a weapon gathers
-You've been out of it, right?#MTLed
+So then I was waiting around thinking about maybe
+going back home when all these scary-looking
+humans with weapons started turning up. So I
+wasn't sure if I'd be able to leave, after all.#RonIffy
```

Comment:
Minor flow:
>_**So, I was waiting around thinking about maybe
going back home, when all these scary-looking
humans with weapons starting turning up. I
wasn't sure I was going to be able to leave...**_

(bit unsure of the last sentence...)